tests/pc/test_po_update.py

```diff
@@ -33,6 +42,18 @@ def ignore_expected_loganalyzer_exceptions(enum_rand_one_per_hwsku_frontend_host
 
     yield
 
+
+@pytest.fixture(scope="function")
+def reload_testbed(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    """
+        Reload dut after test function finished
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    yield None
+    logging.info("Reloading config and restarting swss...")
+    config_reload(duthost)
```

Comment:
You need to wait for critical services to come back after config reload to avoid unstable dut state for next test. Take following example:

https://github.com/Azure/sonic-mgmt/blob/master/tests/dhcp_relay/test_dhcp_relay.py#L237

Reply:
Thanks for the reminder, refined.